### PR TITLE
Fix compatibility issue with typescript@1.7.3

### PR DESCRIPTION
**Pull request by asakusuma**
Fixes https://github.com/tildeio/broccoli-typescript-compiler/issues/7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ function parseOptions(tsconfigPath) {
 
     if (typeof ts.parseConfigFile === 'function') {
       parsedConfig = ts.parseConfigFile(rawConfig.config, ts.sys, path.dirname(tsconfigPath));
+    } else if (typeof ts.parseJsonConfigFileContent === 'function') {
+      parsedConfig = ts.parseJsonConfigFileContent(rawConfig.config, ts.sys, path.dirname(tsconfigPath));
```
Comment (rwjblue) [inline index.js:49]: Can you add a comment just above so we know this is for ~1.7.3?

**Comment by asakusuma** [comment]
Added comment

File: index.js (hunk 1 above)
Comment (asakusuma) [inline index.js:49]: Done

**Comment by rwjblue** [comment]
Released as v0.1.1.

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**